ml/ppo_encoders.py

```diff
@@ -1,40 +1,93 @@
 from abc import abstractmethod
 from typing import Tuple
 
-from ml.typing import State, Value, Policy, ThreeNodesState
+from torch import Tensor
+
 from ml.encoders import TorchEncoder, TowerEncoder
+from ml.functions import concat, euclidean_distance, softmax, sample
+from ml.typing import TensorWithMask
 
 
 class BaseActor(TorchEncoder, config_name='base_actor'):
```

Comment:
Let's take in every forward only one argument `inputs: Dict[str, Any]` and return changed dict at the end